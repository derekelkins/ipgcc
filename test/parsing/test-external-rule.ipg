%declare External %end

S -> External(1, "a");
