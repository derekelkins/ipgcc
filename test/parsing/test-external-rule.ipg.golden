S
  -> External@0(1, "a")[0, EOI];